app/views/name/_form_name.html.erb

```diff
@@ -71,6 +82,7 @@
 
 <p>
   <label for="notes"><%= :form_names_taxonomic_notes.t %>:</label><br/>
+  <span class="mark""><%= :form_names_taxonomic_notes_warning.t %></span>
```

Comment:
The extra double-quote must be a typo here.

Reply:
Good catch. I will fix.

Reply:
I'm so confused.  I still see the extra double-quote.  Did you forget to fix it, or is git still showing the state before the last comment?  If the latter, that's scary, because I've been assuming this diff page on github shows the latest!